src/ItemLinkManagement/Public/New-ItemLink.ps1

```diff
@@ -33,14 +33,21 @@ function New-ItemLink {
                 if (!$item.LinkType) {
                     throw "Existing item '$_path' is not a HardLink, Junction, or SymbolicLink."
                 }
-                if (($item.LinkType -eq $ItemType) -and ($item.Target -eq $_value)) {
-                    "Matching item '$_path' already exists. Skipping" | Write-Verbose
-                    return
+                if (!$Force) {
+                    if (($item.LinkType -eq $ItemType) -and ($item.Target -eq $_value)) {
+                        "Matching item '$_path' already exists. Skipping" | Write-Verbose
+                        return
+                    }
                 }
+                # New-Item -Force does not override an existing junction, hence the need to remove the existing junction
                 if ($ItemType -eq 'Junction') {
-                    if ($item.LinkType -ne $ItemType) {                             # New-Item -Force does not work for junctions, hence the need to remove the existing item
-                        "Itemtype specified as 'Junction'. Removing item '$($item.FullName)' of different item type '$($item.LinkType)'" | Write-Verbose
-                        $item.Delete()                                              # Remove-Item -Force and -Confirm:$false do not suppress confirmation for removal if items exists within symlink or junction target
+                    # Powershell 5 requires a special way to remove a SymbolicLink, see: https://stackoverflow.com/a/63172492
+                    # Remove-Item -Force and -Confirm:$false do not suppress confirmation for removal if items exists within symlink or junction target
+                    if ($PSVersionTable.PSVersion.Major -le 5 -and $item.Attributes -match 'ReparsePoint') {
```

Comment:
@joeltimothyoh refer to this. This removes SymbolicLink and Junction on Windows PSv5.

Reply:
@leojonathanoh THe entire block is only applicable to `junction` items: https://github.com/theohbrothers/ItemLinkManagement/blob/a75fa038137c48fb36834ee87f3d347926db9deb/src/ItemLinkManagement/Public/New-ItemLink.ps1#L42-L52